onnxruntime/test/onnx/runner.h

```diff
@@ -43,6 +43,8 @@ class DataRunner {
   //Time spent in Session::Run. It only make sense when SeqTestRunner was used
   ::onnxruntime::TIME_SPEC spent_time_;
 
+  virtual ~DataRunner();
```

Comment:
>virtual ~DataRunner(); [](start = 2, length = 22)

Extensive comments as to why we need this and people should not make it public.

Reply:
Inherited __dtors should also be private

---
In reply to: [452918182](https://github.com/microsoft/onnxruntime/pull/4475#discussion_r452918182) [](ancestors = 452918182)